TAScheduler/Templates/login.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Login</title>
+    <!-- Add CSS styling here -->
+</head>
+<body>
+    <form id="loginForm" action="/login" method="post">
+        <label for="username">Username:</label>
+        <input type="text" id="username" name="username" required>
+        <label for="password">Password:</label>
+        <input type="password" id="password" name="password" required>
+        <input type="submit" value="Login">
+    </form>
+    <!-- Add validation here -->
```

Comment:
Missing "forgot password" feature - I know this wasn't on the old prototype, but I am going to update that today